### PR TITLE
feat(fetch): adding in the v3/fetch api initial response

### PR DESCRIPTION
**Pull request by bassrock**
## Goal

Make fetch happen.

Add in the /v3/fetch api that is used by Android to load a users list initially.

### CODE REVIEW

File: servers/v3-proxy-api/src/routes/validations/FetchSchema.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      options: {
+        min: 0,
+        max: 5000,
+      },
+    },
+    customSanitizer: {
+      options: (value) => (typeof value === 'string' ? parseInt(value) : value),
+    },
+  },
+  updatedBefore: {
```
Comment (bassrock) [inline servers/v3-proxy-api/src/routes/validations/FetchSchema.ts:79]: Need to implement this in a followup pr..

File: servers/v3-proxy-api/src/routes/validations/FetchSchema.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    isInt: {
+      options: {
+        min: 0,
+      },
+    },
+    customSanitizer: {
+      options: (value) => (typeof value === 'string' ? parseInt(value) : value),
+    },
+  },
+  shares: {
```
Comment (bassrock) [inline servers/v3-proxy-api/src/routes/validations/FetchSchema.ts:90]: Allowing as a value, need to figure out in a followup if we will support

File: servers/v3-proxy-api/src/routes/validations/FetchSchema.ts (hunk 1 above)
Comment (bassrock) [reply servers/v3-proxy-api/src/routes/validations/FetchSchema.ts:79]: https://mozilla-hub.atlassian.net/browse/POCKET-9721

File: servers/v3-proxy-api/src/routes/validations/FetchSchema.ts (hunk 2 above)
Comment (bassrock) [reply servers/v3-proxy-api/src/routes/validations/FetchSchema.ts:90]: https://mozilla-hub.atlassian.net/browse/POCKET-9722

File: servers/v3-proxy-api/src/routes/v3Fetch.integration.ts [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+      {
+        detailType: 'complete',
+        count: '10',
+        offset: '10',
+        tag: 'tag',
+      },
+      {
+        detailType: 'ultra-hd',
+        count: '100000',
+        offset: '-10',
```
Comment (kschelonka) [inline servers/v3-proxy-api/src/routes/v3Fetch.integration.ts:215]: this is valid?

File: servers/v3-proxy-api/src/routes/validations/FetchSchema.ts (hunk 1 above)
Comment (kschelonka) [reply servers/v3-proxy-api/src/routes/validations/FetchSchema.ts:79]: this should be the same as `since` I think, no? can just reuse

**Review by kschelonka — APPROVED** [review]
you go gretchen

**Pull request merged**